src/components/OrganizationSaas/Dashboard/Votings.tsx

```diff
@@ -16,16 +20,32 @@ const Votings = () => {
 
 const VotingsList = () => {
   const { page } = useRoutedPagination()
-  const { data, error, isLoading } = usePaginatedElections(page)
+  const { data, error, isLoading } = usePaginatedElections(page ?? 0)
 
   if (!data) return null
+
   const { elections, pagination } = data
 
   return (
-    <>
+    <Flex flexDirection='column' flexGrow={1} gap='20px'>
       <ProcessesList processes={elections} error={error} loading={isLoading} />
-      {!!elections?.length && <RoutedPagination pagination={pagination} />}
-    </>
+      <Flex
+        mt='auto'
+        justifyContent='end'
+        sx={{
+          '& > div': {
+            display: 'flex',
+            flexDirection: 'column',
+            gap: '5px',
+          },
+          '& > div > div': {
+            ml: 'auto',
+          },
```

Comment:
You could style the Pagination component itself maybe?? I don't think that is a good practice to do this style inheritance when we can just pass props to the Pagination component, but maybe @elboletaire have another opinion

Reply:
Yeah, please, style components with their corresponding theme keys. Stop doing CSS in components with inheritance.